pkg/operator/ceph/file/mds/mds.go

```diff
@@ -100,6 +101,19 @@ func (c *Cluster) Start() error {
 		return errors.Wrap(err, "error checking pod memory")
 	}
 
+	// according to official doc, upgrade mds might need to set max_mds to 1
+	// and stop all stand-by apps first
+	isUpgrade, err := c.isUpgradeMDS()
+	if err != nil {
+		return errors.Wrapf(err, "failed to determine if mds is upgrade")
```

Comment:
```suggestion
		return errors.Wrapf(err, "failed to determine if MDS cluster for filesystem %q needs upgraded", filesystemName)
```

Reply:
@leseb do you think this is a place where we can assume that this IS an upgrade so that we can recover?